.ci_support/linux_aarch64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11numpy2.0python3.10.____cpython.yaml

```diff
@@ -27,7 +27,7 @@ cxx_compiler:
 cxx_compiler_version:
 - '11'
 docker_image:
-- quay.io/condaforge/linux-anvil-cuda:11.8
+- quay.io/condaforge/linux-anvil-x86_64-cuda11.8:ubi8
```

Comment:
But more importantly, those images _do_ have a new enough glibc
```
       virtual packages : __archspec=1=x86_64_v4
                          __conda=24.9.2=0
                          __cuda=11.8=0
                          __glibc=2.28=0
                          __linux=6.5.0=0
                          __unix=0=0
```
so that error makes no sense to me.

Reply:
> Was it just rebuilding the dependency on GLIBC 2.17 

Yes.